Tests/UnitTests/EdgeBridgeTests.swift

```diff
@@ -20,7 +21,19 @@ class EdgeBridgeTests: XCTestCase, AnyCodableAsserts {
     var mockRuntime: TestableExtensionRuntime!
     var edgeBridge: EdgeBridge!
 
+    var fakeSystemInfoService = FakeSystemInfoService()
+
+    var applicationIdentifier: String {
+        return "\(fakeSystemInfoService.applicationName ?? "") \(fakeSystemInfoService.applicationVersionNumber ?? "") (\(fakeSystemInfoService.applicationBuildNumber ?? ""))"
+    }
```

Comment:
looks like you are only using this for default behavior checks where appName/version/buildNo are constant. should we use a constant instead for easy read?